### ISSUE TITLE
Some songs don't download

### ISSUE THREAD

**Issue by long-horse** [Miserlou/SoundScrape#197]
<img width="1306" alt="screen shot 2017-09-05 at 1 15 57 am" src="https://user-images.githubusercontent.com/8702295/30046393-689ca340-91d8-11e7-9879-ea5e317d0a09.png">

scraping my likes downloads a few tracks (which i then unlike) but fails to download a single specific songs that is also in my likes. I don't know what the error means as i dont know python.

**Comment by Benoit-Besnier** [comment]
Yeah, it may happen to me as well. I tried to get "thefatrat - monody" but it was unsuccessful. Well without knowing that much about the code, I only suspect it come from an error while parsing the response coming from SoundCloud. ( JSONDecode is a good piece of information here =) )

**Comment by long-horse** [comment]
youtube-dl had no problem with them but a) its one by one and b) no id3 tags
